### ISSUE TITLE
Not working for 34GN850-B

### ISSUE THREAD

**Issue by Rustle333** [kaleb422/NVapi-write-value-to-monitor#1]
Hello!
Thanks for your work on this. Ive tried this on my LG 34GN850-B and unfortunately it just makes the screen flick black and then return to the original input - much like changing the 0x60 VCP parameter.

Hoping you may be able to help :)

**Comment by kaleb422** [comment]
@ba114 are you able to change any other settings, like brightness, with this or other software like ControlMyMonitor?

**Comment by Rustle333** [comment]
> @ba114 are you able to change any other settings, like brightness, with this or other software like ControlMyMonitor?

Yes works fine to change things like volume, brightness etc with ControlMyMonitor
Just tested, your app also changes volume fine using writeValueToDisplay.exe" 0 0x20 0x62

**Comment by KarenConsoli** [comment]
Hi Everyone! I have exactly the same issue with my LG Ultragear32gp850-b. I can change brightness and volume but when I try to change the input the screen flick black and then return to the original input.

**Comment by kaleb422** [comment]
@ba114 @KarenConsoli  what command line arguments are you using to try change input?
